index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Home page</title>
+</head>
+<body id="body">
+    <section class="header" id="home">
+        <nav class="nav-top-desktop">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav>
+            <div class="nav-top-mobile">
+                <button id="wrapper"><i class="fa-sharp fa-solid fa-bars"></i></button>
+                <img class="img-home" src="images/bioLogo.png" alt="">
+            </div>
+            <div class="nav-wrapper" id="nav-wrapper">
+                <a class="close-button" id="close-wrap"><i class="fa-solid fa-xmark"></i></a>
+                <div>
+                    <a href="">Home</a>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">BC Campaign</a>
+                </div>
+            </div>
+            <div class="nav-wrapper-desktop">
+                <img class="img-home" src="images/bioLogo.png" alt="">
+                <div>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">BC Campaign</a>
+                </div>
+            </div>
+        </nav>
```

Comment:
- [Optional] Please consider wrapping this part of your document in the `<header>` element. This is best for HTML semantics.